core/domain/story_domain.py

```diff
@@ -177,26 +176,69 @@ def create_default_story_node(cls, node_id):
             StoryNode. The StoryNode domain object with default
             value.
         """
-        return cls(node_id, [], [], [], '', '')
+        return cls(node_id, [], [], [], '', None)
```

Comment:
In an older [comment](https://github.com/oppia/oppia/pull/4953#discussion_r190102698) (which wasn't responded to), I asked:

> Btw, here and for the other domain objects, where's the validate() function? It should check things like this (e.g. exploration_id should not be an empty string).

Reply:
Some validations would require other constructs also (like skills). So, thought I would do them after they are done.

Reply:
I'm fine with that going to a separate PR.

Reply:
Ok, will do this later.